### PR TITLE
Add embulk and norikra

### PR DESCRIPTION
**Pull request by yu-iskw**
## Embulk
Embulk is a parallel bulk data loader that helps data transfer between various storages, databases, NoSQL and cloud services.
- https://www.embulk.org/docs/

## Norikra
Norikra is a open source server software provides "Stream Processing" with SQL, written in JRuby, runs on JVM, licensed under GPLv2.
https://norikra.github.io/

### CODE REVIEW

**Comment by mingrammer** [comment]
You should delete the **resources/onprem/data/norikra.png ** and **resources/onprem/data/fluentd.png ** :)

**Comment by mingrammer** [comment]
And how about that move the `embulk` under `onprem.etl` category?

**Comment by yu-iskw** [comment]
That sounds good to me. I will do.

**Comment by mingrammer** [comment]
`resources/onprem/data/norikra.png` is remaining!

**Comment by yu-iskw** [comment]
Thank you for pointing out it. I have removed it.

File: diagrams/onprem/data.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This module is automatically generated by autogen.sh. DO NOT EDIT.
```
Comment (mingrammer) [inline diagrams/onprem/data.py:1]: This file should be deleted manually.

**Comment by mingrammer** [comment]
This file is still remaining .. https://github.com/mingrammer/diagrams/blob/9caaf844de5055f032874bad92153a368281c63c/resources/onprem/data/norikra.png

File: diagrams/onprem/data.py (hunk 1 above)
Comment (yu-iskw) [reply diagrams/onprem/data.py]: Ok, will remove it.

**Comment by mingrammer** [comment]
Screen Shot 2020-04-06 at 6 24 54 PM

**resources/onprem/data/norikra.png** is remaining..!

**Comment by yu-iskw** [comment]
Sorry for not removing the png file. I have removed it.

**Pull request merged**